src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/helpers/AdsComparison.java

```diff
@@ -67,8 +65,9 @@ public boolean compareImageWithScreenshot(final String imageUrl, final WebElemen
       capturedScreen.delete();
       return imageComparison.areBase64StringsTheSame(encodedExpectedScreen, encodedCapturedScreen);
     } catch (IOException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)